plugins/kotlin-as-java/src/test/kotlin/KotlinAsJavaPluginTest.kt

```diff
@@ -53,15 +47,20 @@ class KotlinAsJavaPluginTest : BaseAbstractTest() {
             pagesGenerationStage = { root ->
                 val content = (root.children.single().children.first { it.name == "TestKt" } as ContentPage).content
 
-                val children = content.mainContents.first().cast<ContentGroup>()
-                    .children.filterIsInstance<ContentTable>()
-                    .filter { it.children.isNotEmpty() }
+                val functionRows = content.mainContents.filterIsInstance<ContentTable>()
+                    .single().children
 
-                children.assertCount(2)
+                functionRows.assertCount(6)
```

Comment:
The test is called `top-level functions should be generated`. Because of it, I have changed this behaviour. 

> one for functions, one for properties. If there's no other place that checks that top-level properties are documented,

 Also, this is [KotlinAsJava plugin](https://github.com/Kotlin/dokka/pull/2764/files/e884773487328ca2ec67b547ec70e8f202cee19f#diff-b54c3c69199282583320bd906fc5e76dacb510f082ac5d5e63af85de74236c83) so in Java we have just a public getter for `val testV: Int`. I am not sure that it should check the properties table for a private field.

Reply:
Anyway, I have added a check for properties (just in case).